keepassxc-browser/content/fill.js

```diff
@@ -64,7 +64,7 @@ kpxcFill.fillFromCombination = async function(passOnly) {
     // Set focus to the input field
     field.focus();
 
-    if (kpxc.credentials.length > 1) {
+    if (kpxc.credentials.length > 1 && kpxc.settings.autoCompleteUsernames) {
         // More than one credential -> show autocomplete list
         kpxcUserAutocomplete.showList(field);
     } else {
```

Comment:
I feel like this changes the else condition. I am not familiar with the autoCompleteUsernames option

Reply:
The setting is named a bit poorly, but when it's `true` the Autocomplete Menu is shown when needed. Before this change the filling from combination was prevented if the option was disabled, but actually that needs to be checked later when we are actually trying to show the list.

If the option is disabled and no list is shown, the first credential will be filled.

Reply:
Would it be a big change to change the name of the setting to align with its use?

Reply:
If we change the name, we need to make a function that detects the old setting and transfers the value to the new one. Otherwise old configs are gonna reset the option. So, it's possible.

Reply:
Save that for protocol v2